improver_tests/regrid/test_grid.py

```diff
@@ -152,7 +152,6 @@ class Test_calculate_input_grid_spacing(IrisTest):
     def setUp(self):
         """Set up a lat/lon cube"""
         self.unit = "degrees"
-        self_spacing = 10.0
```

Comment:
@zfan001, hopefully it is fine to remove this?

Reply:
Both self.unit="degree" and "self_spacing =10.0" should be removed.  Just using function's default is okay here.
Thanks